model-engine/tests/unit/conftest.py

```diff
@@ -3290,7 +3290,6 @@ def build_endpoint_request_async_custom(
         model_endpoint_record=ModelEndpointRecord(
             id="test_model_endpoint_id_3",
             name="test_model_endpoint_name_3",
-            created_by=test_api_key,
```

Comment:
why do we need to change this fixture? I'd imagine this may end up breaking various other tests

Reply:
this doesn't need to change, I think this line was deleted accidentally -- undoing now